### PR TITLE
add continuous integration action

### PR DESCRIPTION
**Pull request by ghostwriter**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

- Remove obsolete workflow files.
- Add `continuous-integration.yml`
- Add `.laminas-ci.json` 
- add Build Status and Psalm coverage badges

### CODE REVIEW

File: .laminas-ci.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "extensions": [
+        "bcmath"
+    ],
+    "exclude": [
+        {
+            "name": "PHPUnit on PHP 7.3 with locked dependencies"
```
Comment (Ocramius) [inline .laminas-ci.json]: Hmm, these shouldn't be excluded manually: rather, our `composer.json` declares [`php:^8.0.8`](https://github.com/laminas/automatic-releases/blob/f4a0b35bf35db0b8a66adcf9b557d4709e4b533f/composer.json#L7)

The fact that CS runs with something different looks fishy, potentially being an upstream bug :thinking:

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Comment by Ocramius** [comment]
I've retargeted this to `1.14.x` (next minor) - will need a rebase after checking the issue with manual CI exclusions

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,28 @@
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
 
+## 1.13.1 - TBD
```
Comment (Ocramius) [inline CHANGELOG.md]: We can probably drop `CHANGELOG.md` - it's more messy than useful (especially for merge-ups)

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (ghostwriter) [reply CHANGELOG.md]: Okay.. i was having trouble trying to resign commits originally created by the bot.

Comment (Ocramius) [reply CHANGELOG.md]: I think the entire `CHANGELOG.md` complexity is way too much in this package anyway: we don't need it anymore anyway (although we need to keep it for BC) 

See https://github.com/laminas/technical-steering-committee/blob/4a8ca31d5938a242f071ac6c4f607c07bf88ac95/meetings/minutes/2021-02-01-TSC-Minutes.md#drop-changelogmd-from-repositories-when-it-leads-to-merge-conflicts

Comment (ghostwriter) [reply CHANGELOG.md]: Thanks a rebase from `1.14.x` via git, after retargeting the pr to `1.14.x` on Github, did the trick.

**Review by Ocramius — APPROVED** [review]

**Comment by Ocramius** [comment]
Thanks @ghostwriter!

**Pull request merged**